core/blockchain.go

```diff
@@ -2051,13 +2054,21 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, er
 
 		// Enable prefetching to pull in trie node paths while processing transactions
 		statedb.StartPrefetcher("chain")
-
+		var followupInterrupt uint32
+		// For diff sync, it may fallback to full sync, so we still do prefetch
+		if len(block.Transactions()) >= prefetchTxNumber {
+			throwaway, _ := state.New(parent.Root, bc.stateCache, bc.snaps)
```

Comment:
do we need to swallow the error here

Reply:
actually, it can must be no error, because `statedb` has been initilized in the same way. But I will protect here now.